### PR TITLE
Add ability to change yuidoc options

### LINKED ISSUE #21: Yuidoc config
There is no way to overwrite the [config defaults](https://github.com/cibernox/ember-cli-yuidoc/blob/master/lib/options.js#L39-L45)

I want to change `parseOnly` and i can see why a merge isn't the way to go (preserve `exclude`). My suggestion is:

```js
yuidoc: {
  linkNatives: config.options.linkNatives || true,
  quiet: config.options.quiet || true,
  parseOnly: config.options.parseOnly || false,
  lint: config.options.lint || false,
  exclude: exclusions.join(',')
}
```

### PR DESCRIPTION
**Pull request by fsmanuel**
@cibernox can you review and comment?

closes #21 

ref https://github.com/simplabs/ember-simple-auth/pull/579

### CODE REVIEW

**Comment by cibernox** [comment]
Sorry for the delay. Looks good. Merging

**Comment by cibernox** [comment]
Published to npm in 0.7.1

**Comment by fsmanuel** [comment]
Great!

File: lib/options.js [hunk 1]
```diff
@@ -37,10 +38,10 @@ module.exports = {
       version: getVersion(),
       external: config.external || config.options.external || {},
       yuidoc: {
-        linkNatives: true,
-        quiet: true,
-        parseOnly: false,
-        lint: false,
+        linkNatives: yuidocOptions.linkNatives || true,
```
Comment (rwjblue) [inline lib/options.js]: This prevents `linkNatives: false` from being used (because it would always fall back to `true`).

Comment (rwjblue) [inline lib/options.js]: Should be addressed (along with an error condition if `yuidocOptions` is `undefined`) by https://github.com/cibernox/ember-cli-yuidoc/pull/23.

**Pull request merged**